webapp/run.py

```diff
@@ -1,10 +1,34 @@
 from flask import Flask
+from flask import request
+from flask_api import status
+import web_platform.models as models
+import inflection
 
 app = Flask(__name__)
 
 @app.route('/ping')
 def pong():
     return 'pong'
 
+@app.route('/upload-car-listing', methods=['POST'])
```

Comment:
Bad URL for such an endpoint. See https://nordicapis.com/10-best-practices-for-naming-api-endpoints/